src/js/media-playback-rate-button.js

```diff
@@ -45,10 +45,12 @@ class MediaPlaybackRateButton extends MediaChromeButton {
     if (attrName === 'rates') {
       // This will:
       // 1. parse the space-separated attribute string (standard for representing lists as HTML/CSS values) into an array (of strings)
+      //   The current regex has includes `|,\s*` to preserve legacy behavior for lists with commas.
       // 2. convert that list into numbers (including potentially NaN)
       // 3. filter out all NaNs for invalid values
       // 4. sort the array of numbers to ensure the expected toggle-through order for playback rate.
-      const newRates = (newValue ?? '').split(/,\s?/).map(str => str ? +str : Number.NaN).filter(num => !Number.isNaN(num)).sort();
+      const newRates = (newValue ?? '').trim().split(/\s+|,\s*/)
+        .map(str => Number(str)).filter(num => !Number.isNaN(num)).sort((a, b) => a - b);
```

Comment:
nitpick (non-blocker): may want to reintroduce `.map(str => str ? Number(str) : Number.NaN)` to be "more forgiving" and avoid unintentional 0s.
Example:
```js
newValue = "1 , 2"
(newValue ?? '').trim().split(/\s+|,\s*/).map(str => Number(str)).filter(num => !Number.isNaN(num)).sort((a, b) => a - b)
// result: [0, 1, 2]
(newValue ?? '').trim().split(/\s+|,\s*/).map(str => str ? Number(str) : Number.NaN).filter(num => !Number.isNaN(num)).sort((a, b) => a - b) 
// result: [1, 2]

Reply:
oops, missed this case. I'll take a look.

Reply:
I updated the regex to `/\s*,?\s+/`, which should handle the case you mentioned as well. However, if you think it's still better to bring back the old map, I'm happy to do that if you think that would still be prudent.

Reply:
nope works for me.